apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/DataTransferManagementReceiverImpl.java

```diff
@@ -417,10 +417,11 @@ public MigrateState addShardInfo(CommandParamInfo paramInfo) {
 
         try (Connection sourceConnect = sourceDataSource.begin();
             PreparedStatement preparedInsertStatement = sourceConnect.prepareStatement(
-                    "insert into SHARD (SHARD_ID, SHARD_HASH, SHARD_STATE) values (?, ?, ?)")) {
-            preparedInsertStatement.setLong(1, createdShardId.get());
-            preparedInsertStatement.setBytes(2, paramInfo.getShardHash());
-            preparedInsertStatement.setInt(3, 100); // 100% full shard is present on current node
+                    "UPDATE SHARD SET SHARD_HASH = ?, SHARD_STATE = ? WHERE SHARD_ID = ?")) {
+            preparedInsertStatement.setBytes(1, paramInfo.getShardHash());
+//            preparedInsertStatement.setLong(1, paramInfo.getSnapshotBlockHeight());
+            preparedInsertStatement.setInt(2, 100); // 100% full shard is present on current node
```

Comment:
Move 100 to constants and name it correctly

Reply:
Done